src/visit_vtk/lightweight/vtkVisItUtility.C

```diff
@@ -525,45 +526,45 @@ vtkVisItUtility::FindCell(vtkDataSet *ds, double x[3])
     if (ds->GetDataObjectType() == VTK_RECTILINEAR_GRID)
     {
         int ijk[3];
-        vtkRectilinearGrid *rgrid = (vtkRectilinearGrid*)ds;
+        vtkRectilinearGrid *rgrid = static_cast<vtkRectilinearGrid*>(ds);
         if (ComputeStructuredCoordinates(rgrid, x, ijk) == 0)
             return -1;
-        return rgrid->ComputeCellId(ijk);
+        return static_cast<int>(rgrid->ComputeCellId(ijk));
     }
     else
     {
         // Pulled this from vtkPointSet::FindCell, because for
         // some of our data, their default 'MAXWALK' is too small.
-        int nCells = ds->GetNumberOfCells();
+        vtkIdType nCells = ds->GetNumberOfCells();
         if (nCells == 0)
         {
             return -1;
         }
 
         vtkIdType ptId = 0, cellId = 0;
         vtkCell *cell = NULL;
-        int walk, found = -1, subId;
+        int walk, found = -1, subId=-1;
```

Comment:
why init to -1?

Reply:
I figured that was a good error condition value.

Reply:
Sounds good. I was just wondering if changing it to be initialized to -1 could change anything down the road, but it sounds like you thought about that.